akka-stream-tests/src/test/scala/akka/stream/io/FileSinkSpec.scala

```diff
@@ -222,6 +223,13 @@ class FileSinkSpec extends StreamSpec(UnboundedMailboxConfig) {
         checkFileContents(f, TestLines.takeWhile(!_.contains('b')).mkString(""))
       }
     }
+
+    "complete with failure when file cannot be open" in {
+      val completion = Source.single(ByteString("42"))
+        .runWith(FileIO.toPath(Paths.get("/I/hope/this/file/doesnt/exist.txt")))
```

Comment:
Likely fast, but just for good measure, can we get the path from the in-memory fs that the other tests use instead of touching actually touching disk?